access-control-app/src/cmr/access_control/data/acl_json_results_handler.clj

```diff
@@ -64,9 +64,7 @@
 
 (defmethod elastic-results/elastic-result->query-result-item [:acl :json]
   [context query elastic-result]
-  (let [item (->> elastic-result
-                  :fields
-                  (util/map-values first))
+  (let [item (:_source elastic-result)
```

Comment:
Nitpick: name this instance of `item` something like `source` or `field-values` like you have in the file below

Reply:
My changes were not related to naming of the variables either here or below. I will rename it to `source`.